monai/losses/unified_focal_loss.py

```
@@ -226,6 +251,19 @@ def forward(self, y_pred: torch.Tensor, y_true: torch.Tensor) -> torch.Tensor:
             else:
                 y_true = one_hot(y_true, num_classes=n_pred_ch)
 
+        if not self.include_background:
+            if n_pred_ch == 1:
+                warnings.warn("single channel prediction, `include_background=False` ignored.")
```

Ruff (0.14.3)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)